plone/app/iterate/browser/control.py

```diff
@@ -29,7 +29,7 @@
 from plone.memoize.view import memoize
 from Products.Five.browser import BrowserView
 
-import Products.CMFCore.permissions
+from .. import permissions
```

Comment:
Does this work as you expect in Python 3?

Reply:
Why wouldn't it, @Rotonen?

Reply:
.. is fine

Reply:
Is fine, but it way better to just use ``from plone.app.iterate import permissions`` is not that much typing and you can keep imports sanely

Reply:
I disagree that absolute imports are way better.  Relative imports do save some minor refactoring time when package names or structure changes, and if you're typing your imports out they do save some time though you should not be typing your imports out, but the far greater value is when *reading* code.  They make it very clear which dependencies of the code are internal to the package and external to the package even when just skimming the code.  This is especially the case for a large ecosystem of packages such as Plone where a developer may be working on 3-6 `plone.app.*` packages at the same time and it adds cognitive burden to require the developer to remember *which* `plone.app.*` package they're reading ATM in order to determine which code dependencies are internal.  The core language and the community that develops it, provide this feature for well considered reasons.